tests/backend/checkmach/test_assume.ml

```diff
@@ -0,0 +1,2 @@
+let[@assume noalloc][@inline never] test1 n = (n,n)
```

Comment:
(For coverage, maybe also have
a test using that `test1` function
from another file.)